openpype/hosts/blender/api/ops.py

```diff
@@ -350,18 +354,19 @@ class BuildWorkFile(bpy.types.Operator):
 
     bl_idname = "wm.avalon_builder"
     bl_label = "Build First Workfile"
+    # bl_property = "should_save_first_workfile"
     _app: QtWidgets.QApplication
 
+    # Should save property
+    should_save: bpy.props.BoolProperty(name="Save as...", default=True)
```

Comment:
I think it is more explicit and consistent to name it `save_as`.

Reply:
Change applied!